docs/release/trg-0/trg-7-4.md

```diff
@@ -0,0 +1,94 @@
+
+This page contains information about legal obligations and checks of your 3rd party content. The reference is always the [Eclipse Foundation Project Handbook](https://www.eclipse.org/projects/handbook/#ip-third-party).
+
+:::info
+
+The requirements described here must be met for each contribution.
+
+:::
+
+## Description
+
+The term third-party content refers to any content that is leveraged by the Eclipse project that is not produced and/or maintained as an asset of the project. This includes content from other Eclipse projects. See the complete explanation of [third-party content](https://www.eclipse.org/projects/handbook/#ip-third-party).
+
+All third-party content has to be checked and approved by the Eclipse Foundation IP Team. There are two ways:
+
+- Creating an IP issue manually (e.g. fonts, images, ...);
+- Using the Eclipse Dash License Tool to creat IP issues in an automated way (libraries).
+
+All third party content used has to be documented in the NOTICE file or the DEPENDENCY file. (TODO AWI: add link to TRG 7.01)
+
+:::info
+
+Only project committers can request IP issues, manually or via the Dash Tool!
+
+:::
+
+### Checking libraries using the Eclipse Dash License Tool
+
+You can request the status of your used libraries via the [Dash Licence Tool](https://github.com/eclipse/dash-licenses/blob/master/README.md), see also the [handbook](https://www.eclipse.org/projects/handbook/#ip-license-tool).
+
+**Steps:**
+
+- Download the lastest version: README => Get It => Download Link
+- For every repository / technology:
+  - Create the list of transitive dependencies, see the README
+  - Run the Dash Tool with the parameters "-project automotive.tractusx" and "-summary DEPENDENCIES" (or use the maven plugin)
+- Check for libraries with status "rejected" in the DEPENDENCIES file, if present, the dependency cannot be used and has to be removed.
+- Check for libraries with status "restricted", if so request checks by automatically creating issues via the Dash Tool:
+  - include the -review option;
+  - pass the API token via the -token option; and
+  - pass the project id via the -project option.
+- Create an issue in YOUR repository with the links to the IP Lab issues, [Example](https://github.com/eclipse-tractusx/daps-registration-service/issues/28)
+- Track your [issues](https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues?search=automotive.tractusx&sort=created_date&state=opened)
+- Provide support if an issue is labeled with "Help wanted"
+- Add the summary as DEPENDENCY file to the according repository (root level)
+
+**Important Notes:**
+
+- Get your API Token (see README of the Dash Tool), note that only committers can get an API Token
+- Do **NOT** share your API Token!
+- DO **NOT** integrate into automatic builds the "Automatic IP Team Review Requests" (creation of ip tickets: " -review option") via your personal API Token
+To integrate in your automatic builds: request a dash-bot via the EF Helpdesk.
```

Comment:
Perhaps putting this in the line above like this: " API Token (request a dash-bot via the EF Helpdesk)